dashx/src/main/java/com/dashx/sdk/DashXClient.kt

```diff
@@ -150,18 +171,13 @@ class DashXClient {
             .build()
     }
 
-    fun generateAnonymousUid(regenerate: Boolean = false) {
-        val dashXSharedPreferences: SharedPreferences =
-            getDashXSharedPreferences(context!!)
-        val anonymousUid =
-            dashXSharedPreferences.getString(SHARED_PREFERENCES_KEY_ANONYMOUS_UID, null)
+    fun generateAccountAnonymousUid(regenerate: Boolean = false):String {
```

Comment:
Remove `regenerate` from this function, and remove the interaction with shared prefs within this method.